chat-plugins/info.js

```diff
@@ -202,7 +203,10 @@ exports.commands = {
 				}
 			});
 		}
-		if (!results.length) return this.errorReply("No results found.");
+		if (!results.length) {
+			let invalidTarget = net.isIP(target.trim()) === 0 && !target.includes('.');
+			return this.errorReply("No results found" + (invalidTarget ? " because this is not a valid IP or host search quarry." : "."));
```

Comment:
You probably don't need the "No results found because" text if the passed target was invalid.